website/content/api-docs/secret/pki.mdx

```diff
@@ -696,19 +696,30 @@ have access.**
 - `csr` `(string: <required>)` - Specifies the PEM-encoded CSR.
 
 - `key_usage` `(list: ["DigitalSignature", "KeyAgreement", "KeyEncipherment"])` -
-  Specifies the allowed key usage constraint on issued certificates. Valid
+  Specifies the default key usage constraint on the issued certificate. Valid
   values can be found at https://golang.org/pkg/crypto/x509/#KeyUsage - simply
   drop the `KeyUsage` part of the value. Values are not case-sensitive. To
   specify no key usage constraints, set this to an empty list.
```

Comment:
Should we also remove this sentence in both field definitions? `To specify no key usage constraints, set this to an empty list.`

Reply:
Being pedantic, but the [KeyUsage](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.3) field is a constraint ("restriction" in the RFC) on the certificate (and we call it the KU constraint above). 

How about "To specify no _default_ key usage constraints, ..."?

Reply:
Yes that sounds good